RPIPs/RPIP-60.md

```diff
@@ -20,6 +20,14 @@ The security council may trigger a faster upgrade with a shorter delay, but ther
 
 Additionally the security council may veto contract upgrades. The veto power should not be used lightly, and misuse may result in replacement of the security council by the pDAO.
 
+## Motivation
+
+This RPIP is motivated by the desire to protect participants in the Rocket Pool as best as possible from unfavorable upgrades. Participants should have time to exit the protocol if it changes in a way they do not agree with, _prior_ to those changes coming into effect. 
```

Comment:
"participants in the Rocket Pool" -> "Node Operators"?

Reply:
Thanks, typo. Holdover from when it was 'the protocol'